test/addons/force-context-aware/binding.cc

```diff
@@ -0,0 +1,14 @@
+#include <node.h>
+#include <v8.h>
+
+void Method(const v8::FunctionCallbackInfo<v8::Value>& args) {
+  v8::Isolate* isolate = args.GetIsolate();
+  args.GetReturnValue().Set(v8::String::NewFromUtf8(
+        isolate, "world", v8::NewStringType::kNormal).ToLocalChecked());
+}
+
+void init(v8::Local<v8::Object> exports) {
+  NODE_SET_METHOD(exports, "hello", Method);
```

Comment:
You don't really need to export anything since it's not going to be loaded anyway.